docs/start/configure-qiskit-local.mdx

```diff
@@ -52,6 +52,9 @@ Set these environment variables to alter the default behavior of Qiskit:
 
 ## Next steps
 
-<Admonition type="tip" title="Recommendation">
-    Follow the steps in [Hello world](hello-world) to write and run a quantum program.
+<Admonition type="tip" title="Recommendations">
+  - Read the [contributing guidelines](https://github.com/Qiskit/qiskit/blob/main/CONTRIBUTING.md) to contribute to the open-source Qiskit SDK.
```

Comment:
Should this be lower in the list? Or maybe less strong, like 

> If interested in contributing, read the contributing guidelines....

We don't want to imply you need to contribute.

Reply:
or maybe 

> - Read the [contributing guidelines](https://github.com/Qiskit/qiskit/blob/main/CONTRIBUTING.md) if you want to contribute to the open-source Qiskit SDK.
?

Reply:
I'll do both - thanks for the input!